packages/botonic-dx/sample-config/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "scripts": {
+    "build": "tsc",
+    "build-ci": "tsc -b tsconfig.tests.json",
+
+    "lint": "npm run lint-core -- --fix",
+    "lint-ci": "npm run lint-core -- -c ../.eslintrc-ci.js",
+    "lint-core": "eslint_d --cache --quiet '*.js' 'src/**/*.js*' 'src/**/*.ts*' 'tests/**/*.ts*'",
```

Comment:
Is it needed to put here the files we want to lint if in `tsconfig.eslint.json` we're already doing it?

Reply:
yes, it's a limitation of eslint